devito/arguments.py

```diff
@@ -79,6 +78,10 @@ def __init__(self, name, provider, reducer, default_value=None):
         super(ScalarArgument, self).__init__(name, provider, default_value)
         self.reducer = reducer
 
+    @property
+    def as_symbol(self):
+        return Symbol(self.name)
```

Comment:
This is ``master`` already, so this needs rebasing?

Reply:
Fixed (confirm)